test/component/CustomsDeclarationAmendSpec.scala

```diff
@@ -118,7 +118,7 @@ class CustomsDeclarationAmendSpec extends ComponentTestSpec with AuditService wi
       status(result) shouldBe ACCEPTED
 
       And("the response body is empty")
-      contentAsString(result) shouldBe 'empty
+      contentAsString(result) shouldBe Symbol("empty")
```

Comment:
We should use ScalaTest's `org.scalatest.matchers.dsl.MatcherWords#empty` here instead.

It's already in scope and Symbol is already deprecated: https://docs.scala-lang.org/scala3/guides/migration/incompat-dropped-features.html